doc/users/next_whats_new/rcparams_legend.rst

```diff
@@ -0,0 +1,26 @@
+New rcParams for ledend: set legend labelcolor globaly
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The new :rc:`legend.labelcolor` allows toggling valid color string(for example, 'black'),
+versus choosing to match the color of the line of marker using 'linecolor',
+'markerfacecolor'(or 'mfc'), or 'markeredgecolor' (or 'mec').
+
+.. plot::
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+    
+    # Make some fake data.
+    a = np.arange(0, 3, .02)
+    c = np.exp(a)
+    d = c[::-1]
+    plt.rcParams['legend.labelcolor'] = 'linecolor'
+
+
```

Comment:
This is the important line, so I would put it first.

Also, only one blank line.